WordPress/src/main/java/org/wordpress/android/ui/stats/StatsAbstractListFragment.java

```diff
@@ -148,6 +150,8 @@ public void onSaveInstanceState(Bundle outState) {
             }
         }
 
+        outState.putParcelable(ARGS_EXPANDED_ROWS, new SparseBooleanArrayParcelable(mGroupIdToExpandedMap));
```

Comment:
I think most of the time `mGroupIdToExpandedMap` will be empty, so it would make sense to only save it here if it's not empty and then only restore it if the key exists.